main.py

```diff
@@ -99,11 +104,9 @@ def setup_target_task_training(args, target_tasks, model, strict):
 
     Returns
     ----------------
-    task_names_to_avoid_loading: list of strings, if we don't allow for
```

Comment:
Don't we still need to return it? If we're loading the same checkpoint here and in reload_model, we'll need to do the same filtering.

Reply:
I actually realize we don't need it. In the load_model_for_target_train_run function we can just pass in the specific task we're finetuning on into the skip_task_models param because we're only going to be using the task specific module for that task in that specifc target task run anyways.